substrate/frame/staking/src/benchmarking.rs

```diff
@@ -598,8 +602,8 @@ mod benchmarks {
 	}
 
 	#[benchmark]
-	// Worst case scenario, the list of invulnerables is very long.
-	fn set_invulnerables(v: Linear<0, { MaxValidators::<T>::get() }>) {
+	// Worst case scenario according to the invulnerables limit.
```

Comment:
the comment lost its main message I think,  the comment was saying that the list is very long in worst case shouldn't be negliged.

Reply:
Removed in https://github.com/paritytech/polkadot-sdk/pull/6445/commits/c881bb0015cfd0c9248ebd53f751ff267b5ba5d9